models/image.py

```diff
@@ -340,285 +341,66 @@ def mid_mjd(self):
-        # the exposures associated with each image,
-        # and the images recursively associated with other images
-        stmt = sa.select(columns)
-        stmt = first_table.add_to_joins(stmt)
-        stmt = first_table.add_children_to_joins(stmt)
-
-        # since we aggregate every associated image/exposure using bit_or,
-        # we need to make sure to group by the top level image and exposure combination
-        stmt = stmt.group_by(first_table.img.id, first_table.exp.id)
-
-        # this last where makes sure that when querying on Image externally
-        # each Image row will correspond to one result, hence the scalar_subquery
-        stmt = stmt.where(cls.id == first_table.img.id).scalar_subquery()
-
-        return stmt
-
-    @bitflag.inplace.setter
-    def bitflag(self, value):
-        allowed_bits = 0
-        for i in image_badness_inverse.values():
-            allowed_bits += 2 ** i
-        if value & ~allowed_bits != 0:
-            raise ValueError(f'Bitflag value {bin(value)} has bits set that are not allowed.')
-        self._bitflag = value
+    lim_mag_estimate = sa.Column(
+        sa.Float,
+        nullable=True,
+        index=True,
+        doc=(
+            'Limiting magnitude estimate for the image, '
+            'from the first time the image was processed.'
+        )
+    )
 
-    @property
-    def badness(self):
-        """
-        A comma separated string of keywords describing
-        why this data is not good, based on the bitflag.
-        This includes all the reasons this data is bad,
-        including the parent data models that were used
-        to create this data (e.g., the Exposure underlying
-        the Image).
-        """
-        return bitflag_to_string(self.bitflag, data_badness_dict)
-
-    @badness.setter
-    def badness(self, value):
-        """Set the badness for this image using a comma separated string. """
-        self.bitflag = string_to_bitflag(value, image_badness_inverse)
-
-    def append_badness(self, value):
-        """Add some keywords (in a comma separated string)
-        describing what is bad about this image.
-        The keywords will be added to the list "badness"
-        and the bitflag for this image will be updated accordingly.
-        """
-        self.bitflag |= string_to_bitflag(value, image_badness_inverse)
+    bkg_mean_estimate = sa.Column(
+        sa.Float,
+        nullable=True,
+        index=True,
+        doc=(
+            'Background estimate for the image, '
+            'from the first time the image was processed.'
+        )
+    )
 
-    description = sa.Column(
-        sa.Text,
+    bkg_rms_estimate = sa.Column(
+        sa.Float,
         nullable=True,
-        doc='Free text comment about this image, e.g., why it is bad. '
+        index=True,
+        doc=(
+            'Background RMS estimate for the image, '
+            'from the first time the image was processed.'
+        )
```

Comment:
These two fields should be set in preprocessing.py (go to line ~265, which is where the values are extracted).

Reply:
I will do the actual filling in of these fields in a separate PR.